gnc/sub8_perception/src/sub8_pcl/cv_param_helpers.cc

```diff
@@ -0,0 +1,30 @@
+#include <sub8_pcl/cv_tools.hpp>
+#include <ros/console.h>
+
+namespace sub {
+
+/// Range is an output
+void range_from_param(std::string &param_root, Range &range) {
+  /// Ex: param_name = /buoy/red --
+
+  // No overhead in doing this
+  ros::NodeHandle nh;
+  ROS_WARN("-- Waiting for parameter %s, (No timeout)", param_root.c_str());
```

Comment:
Gotcha. I think this is closer to a warning, because the whole buoy detector will hang on this until the param exists. The user should be warned that the node might stop operating